addons/website_event/models/event_event.py

```diff
@@ -42,8 +42,17 @@ def _default_question_ids(self):
     # description
     subtitle = fields.Char('Event Subtitle', translate=True)
     # registration
-    is_participating = fields.Boolean("Is Participating", compute="_compute_is_participating")
+    is_participating = fields.Boolean("Is Participating", compute="_compute_is_participating",
+                                      search="_search_is_participating")
     # website
+    can_see_on_website = fields.Boolean(compute='_compute_can_see_on_website', search='_search_can_see_on_website')
+    full_website_url = fields.Char('Full Website Url', compute='_compute_full_website_url',
```

Comment:
Actually it looks to be the event register URL, could rename to "event_register_url" ?